### ISSUE TITLE
Make it possible to publish an ontology so it can be accessed publicly through an URL (unzipped)

### ISSUE THREAD

**Issue by rogargon** [protegeproject/webprotege#717]
WebProtege lacks nice visualization features so my workflow is to download the ontology as a ttl.zip file, unzip, upload it and then point tools like WebOWL to be able to share interactive visualizations of the ontology for others to explore.

It would be really useful to be able to "publish" the ontology so it is available through an URL like:
https://webprotege.stanford.edu/download?project=<project-code>&format=ttl

This is almost possible right now using the same URL that is used to download the ontology. However, it seems to make the ontology available just zipped. This way, I cannot use that link to load it into WebOWL or other tools directly from the URL.

Moreover, the user performing the visualization needs to have a WebProtege account, be logged in and have the ontology shared (directly or by link). Therefore, the other missing piece would be to, optionally, share the ontology code to anyone with the link.

With all these, which does not seem possible right now, it would be very straightforward to augment WebProtege with nice visualizations like WebOWL.

**Comment by anupam1050** [comment]
yes, it would be great to let user publish the ontology and visualize using WebVOWL or is it possible to integrate the WebVOWL as another tab in WebProtege . Integration of WebVOWL with WebProtege will make this Complete with rich Visualization.

**Comment by anupam1050** [comment]
@rogargon  we have done a work around in Webprotégé code to publish the Ontology to Azure blob storage and then written a python code to generate complete webVOWL URL based on user specific files that a user you can directly open in webVOWL.

**Comment by rogargon** [comment]
Interesting @anupam1050 Is this available in the last version of WebProtege or you added this feature to a custom deployment?

**Comment by anupam1050** [comment]
Hi,

We have enabled this as a custom deployment.

Thanks
Anupam

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by smithGit** [comment]
Good day, @anupam1050 from Michigan:  Have you activated the system to deploy ontologies from the Azure blob storage?  Tat would be pretty cool.  What is the best way to publish an ontology in development prior to posting on an official, curated ontology system?  Can it now be done via Github?
